scripts/build-dist.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
-set -ex
+set -Exu
```

Comment:
https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html#The-Set-Builtin
>If set, any trap on ERR is inherited by shell functions, command substitutions, and commands executed in a subshell environment. The ERR trap is normally not inherited in such cases.

I'm not sure if it makes a huge difference but looked safer to me since it covers more areas.